### ISSUE TITLE
Docker image update

### ISSUE THREAD

**Issue by kortenkamp** [calcom/docker#308]
I am a bit confused about the way to update the calcom docker image. I used a custom build in my first install a long while ago, adapted the `docker-compose.yaml` to work the rest of my web services, and since then I used `docker compose pull` to update the image (and of course, restarting it ). 

The `docker-compose.yaml` contains an image location for the calcom docker image:
```
    image: calcom.docker.scarf.sh/calcom/cal.com
```

So... I assume that my initial custom build is no longer needed, as the image is being pulled from there. However, the image available there is version 3.4.7-sh, not 3.5.0, as announced in the releases here.

Also, using docker to search for calom shows
```
# docker search calcom
NAME                       DESCRIPTION                                     STARS     OFFICIAL   AUTOMATED
calcom/cal.com             The official and verified Cal.com Docker Ima…   17
…
```

which is apparently the 3.4.7 image that I pull. 

My question is: Why is the official image behind the release on calcom/docker, while the release 3.5.0 is announced here? And what could I do to be on the latest image? And finally – is my clone of calcom/docker need at all? Would it be anough to just reference `image: calcom/cal.com` in my `docker-compose.yaml`?

Maybe these are stupid questions, so I would like to apologize for them in advance, just in case.

**Comment by kortenkamp** [comment]
Apparently I am not the only person who is confused – #267 (closed) and #239 (still open!) mention similar problems. Can somebody clarify and – maybe – update the `README.md`?

**Comment by krumware** [comment]
It looks like there are a couple of missing dependencies in core calcom, introduced in a recent version. The calcom team has been notified. I'll work to get those images pushed when that is resolved, and also shift the release tagging to occur after the image has been built successfully.

**Comment by krumware** [comment]
new issue ref: https://github.com/calcom/cal.com/issues/12446

**Comment by krumware** [comment]
another new issue ref: https://github.com/calcom/cal.com/issues/12497

**Comment by krumware** [comment]
The above issues have been closed and the dependency issues resolved. v3.5.2 is now available.  I've removed the missing releases that were failing due to the dependency issues. We'll shift our attention to preventing the release tagging on unsuccessful builds.

Also, there are still active conversations with the core team about removing build variable dependencies.
Also asking about whether the scarf endpoint needs to be kept. (@kortenkamp, to answer your question, its a statistics tool and just logs forwards requests to the calcom/cal.com dockerhub, so it's acceptable to use calcom/cal.com)
